scout/run.py

```diff
@@ -5415,10 +7255,67 @@ def main(opts: argparse.NameSpace):  # noqa: F821
                           "and that all active measure names match those " +
                           "found in the 'name' field for corresponding " +
                           "measure definitions in ./ecm_definitions"))
+    # Further check to ensure that no active measures are tagged for removal
+    meas_summary_restrict = [
+        m for m in meas_summary if m["name"] in active_meas_all and
+        m["remove"] is False]
+    if len(meas_summary_restrict) == 0:
+        raise ValueError(
+            "Active measures were found but all tagged for removal.")
+
+    # Set flag for fuel splits
+    split_fuel = (meas_summary_restrict[0]["usr_opts"]["split_fuel"] is True)
+
+    # Set a flag for detailed breakouts
+    if (meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '1' and split_fuel is True):
+        brkout = "detail"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '2':
+        brkout = "detail_reg"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '3':
+        brkout = "detail_bldg"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '4':
+        brkout = "detail_fuel"
+    elif (meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '5' or (
+            meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '1' and
+            split_fuel is not True)):
+        brkout = "detail_reg_bldg"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '6':
+        brkout = "detail_reg_fuel"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '7':
+        brkout = "detail_bldg_fuel"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '8':
+        brkout = "detail_reg_fuel_codesbps"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '9':
+        brkout = "detail_codesbps"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '10':
+        brkout = "detail_reg_codesbps"
+    elif meas_summary_restrict[0]["usr_opts"]["detail_brkout"] == '11':
+        brkout = "detail_codesbps_fuel"
     else:
-        measures_objlist = [
-            Measure(handyvars, **m) for m in meas_summary if
-            m["name"] in active_meas_all and m["remove"] is False]
+        brkout = "basic"
```

Comment:
Can use a dict for most of these instead of lots of if statements, eg `brkout_lookup = {"2": "detail_reg", "3": "detail_bldg"...}`